plugins/reference-assets/android/src/androidTest/java/com/intuit/player/android/reference/assets/action/ActionTest.kt

```diff
@@ -43,12 +44,12 @@ class ActionTest : AssetTest("action") {
     fun transitionToEndSuccess() {
         launchMock("transition-to-end")
 
-        currentView.shouldBeView<LinearLayout> {
-            assertEquals(2, childCount)
-            get(0).shouldBeView<Button> {
-                assertEquals("End the flow (success)", text.toString())
-                performClick()
-            }
+        val collectionValues = currentView?.findViewById<LinearLayout>(R.id.collection_values) ?: throw AssertionError("current view is null")
```

Comment:
this change and the text one, are they due to having the assets in a collection?

Reply:
yes. previous` currentView.shouldBeView<LinearLayout>` failed because the root/parent is now ConstraintLayout due to the new XML view